.travis.yml

```diff
@@ -30,28 +30,38 @@ apt_sources: &apt_sources
   - sourceline: "ppa:jamesw05/tmux"
     key_url: "https://keyserver.ubuntu.com/pks/lookup?op=get&search=0x51a89e9c35f79436"
 
+common_packages: &common_packages
```

Comment:
This is unrelated really. I guess it cleans things up a bit. Can also remove it again if you think ...